analytics/tests/test_support_views.py

```diff
@@ -1,8 +1,12 @@
 from datetime import datetime, timedelta, timezone
+from typing import TYPE_CHECKING
 from unittest import mock
 
 import orjson
-from django.http import HttpResponse
+
+if TYPE_CHECKING:
+    from django.test.client import _MonkeyPatchedWSGIResponse
+
 from django.utils.timezone import now as timezone_now
```

Comment:
Move these `if TYPE_CHECKING:` blocks below all our normal imports, to avoid splitting isort groups in half.